### PR TITLE
Replace Vagrant with Docker in the SWF

### PR DESCRIPTION
**Pull request by tobsan**
This follows from a change made recently in `pelux-manifests`

### CODE REVIEW

**Review by rpannek — APPROVED** [review]
Looks really nice and clean now!

**Comment by tobsan** [comment]
@rpannek Yeah, I figured it is easier to just say "use docker but act as if building locally" instead of giving super detailed instructions for the docker case.

File: docs/chapters/baseplatform/building-PELUX-sources.rst [hunk 1]
```diff
@@ -74,17 +74,16 @@ Intel NUC are ``.wic`` images, which are complete disk-images. For the Raspberry
...
 the SD card.
 
-Building with Vagrant
----------------------
+Building with Docker
+--------------------
 
-When we build internally at Pelagicore in our CI system, we use vagrant.  Please
+When we build internally at Pelagicore in our CI system, we use Docker.  Please
 note that we only run this setup in a GNU/Linux system at Pelagicore. It should
```
Comment (thenor) [inline docs/chapters/baseplatform/building-PELUX-sources.rst]: This line is a bit repetitive. I would suggest taking the opportunity to remove the seconds line's "at Pelagicore".

Comment (tobsan) [reply docs/chapters/baseplatform/building-PELUX-sources.rst]: Fixed!

**Review by thenor — APPROVED** [review]
Changes look good.
When I check the document the "Note" doesn't look good, but maybe it looks better in the generated docs.

**Pull request merged**